test/services/teams/helper/helper.hook.js

```diff
@@ -15,7 +15,7 @@ const _DefaultServiceOptions = {
 
 const _DefaultHeaderParams = {
 	authorization: '<jwtToken>',
-	host: process.env.HOST || 'localhost:3030',
+	host: HOST,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['HOST' is not defined.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5246313)